lib/worker/subprocess.js

```diff
@@ -118,6 +117,10 @@ ipc.options.then(options => {
 			} catch (_) {}
 		}
 
+		// Install dependency tracker after the require configuration has been evaluated
+		// to make sure we also track dependencies with custom require hooks
+		dependencyTracking.install(testPath);
```

Comment:
Since we already know the files, we could just call `dependencyTracking.track` for every entry in `options.require`, right?

Reply:
Actually I’m not really sure if this is even needed. If I want my files in `options.require` being considered in watch mode I have to add a pattern to `options.sources` for them. And if I modify one of the files in `options.require` then ava needs to rerun all tests because they are required in every test file, so there is no need to determine in which test files they have been used.
I’ve implement an integration test for this use-case and I think it already works as expected.

Reply:
Oh well reasoned, nice, that keeps things simple.